README.md

```diff
@@ -146,15 +166,17 @@ Currently all aggregation types are compatible with all aggregation windows. Lat
     lastNSamples := 100
     precisionSubsets := 10
     wnd2 := stats.NewWindowSlidingCount(lastNSamples, precisionSubsets)
-    
     wn3 := stats.NewWindowCumulative()
 
 
 ### To creater/register a view
```

Comment:
If you can edit this while you are there, it would be great.

"To  create/register a view"